### ISSUE TITLE
Module "site" : Affichage du formulaire de saisie dans le volet latéral

### ISSUE THREAD

**Issue by xavyeah39** [PnX-SI/GeoNature-citizen#417]
### What is the bug or the crash?

La répercution du paramètre `NEW_OBS_FORM_MODAL_VERSION: false` sur les programmes de type "site" n'est pas fonctionnelle et intégrée differement que pour les programmes de type "observation".

Observation (OK) : 
![image](https://github.com/PnX-SI/GeoNature-citizen/assets/16793813/a802ffad-acf0-491c-b57f-2fbe235b011f)

Site : 
* La saisie de la géométrie ne se fait pas depuis le composant de carte principale de la page "programme"
* L'intégration du formulaire dans le volet latéral gauche (list) n'est pas top

![image](https://github.com/PnX-SI/GeoNature-citizen/assets/16793813/375cc9ec-5de6-4ab4-b36d-11d23008e65b)

### Steps to reproduce the issue

Définir le paramètre `NEW_OBS_FORM_MODAL_VERSION` à `false` dans le fichier `./frontend/src/conf/app.config.ts` et créer un programme de type site/visites.

### Versions

v1.1.0

**Comment by camillemonchicourt** [comment]
Il faudrait factoriser les formulaires, sinon ils sont toujours différents/divergents.

**Comment by hypsug0** [comment]
Oui, se pose aussi la question de la pertinence de cette customization de l'emplacement du formulaire. Cela ajoute de la complexité à la maintenance de l'outil. Personnellement, je serai pour supprimer ce option pour ne conserver que la modale.

**Comment by camillemonchicourt** [comment]
Oui c'est toujours galère de maintenir et de gérer différents cas, différentes structures de pages avec des options...
Mais ce développement a été financé et validé (https://github.com/PnX-SI/GeoNature-citizen/issues/218), donc sauf si il n'a aucun intérêt et aucun usage, c'est compliqué de le supprimer après coup... 🤔
